digdag-docs/src/operators/py.md

```diff
@@ -20,3 +20,12 @@ See [Python API documents](../../python_api.html) for details including variable
   ```
   py>: tasks.MyWorkflow.my_task
   ```
+
+The python defaults to `python`. If an alternate python such as `/opt/conda/bin/python` is desired, use the `python` option in the `_export` section.
+
+    _export:
+      py:
+        python: ["/opt/conda/bin/python"]
```

Comment:
It should be String instead of Array for `python` option. Otherwise, we get configuration error.

```
2018-10-03 10:40:10 -0700 [INFO] (XNIO-1 task-7): Starting a new session project id=1 workflow name=py-test session_time=2018-10-03T17:40:09+00:00
2018-10-03 10:40:11 -0700 [INFO] (0032@[0:pyop-test]+py-test+run): py>: simple_task.run
2018-10-03 10:40:11 -0700 [ERROR] (0032@[0:pyop-test]+py-test+run): Configuration error at task +py-test+run: Expected string type for key 'python' but got ["/opt/conda/bin/python3"] (array) (config)
> Can not deserialize instance of java.lang.String out of START_ARRAY token
 at [Source: N/A; line: -1, column: -1] (json mapping)
2018-10-03 10:40:11 -0700 [INFO] (0032@[0:pyop-test]+py-test^failure-alert): type: notify
```

Reply:
Thank you! should fix the documentation.